### ISSUE TITLE
The MiG-29 appears to be broken

### ISSUE THREAD

**Issue by swampmo** [FGMEMBERS/MiG-29#2]
Someone else can try but All I get is a glider and yes I deleted the -master. This could be due to the last commit.

**Comment by JMaverick16** [comment]
Mmm, do you know that you shouldn't pull down completely the throttle otherwise the engines shut down? At least this is what I get.

**Comment by FGDATA** [comment]
@swampmo 

can you test this for us, please?

in the MiG-29 git repo

          git checkout -b test 0bedc

and then launch again. To verify if the last commit is creating a problem

**Comment by JWocky** [comment]
I had a look at MiG-29s too and found two, Mig-29 and Mig-29_9-12. The 29_9-12 had that throttle behavior, the Yasim one, the Mig-29 without anything behind it didn't even spawn on the runway but way behind it in the dirt, so there is a lot more wrong with that one.

**Comment by swampmo** [comment]
This is old Mig-29 not one made by Mercury Mercenary, which is the one that full back throttle kills the engine. That model is fine I flew it last night.

**Comment by swampmo** [comment]
@FGDATA  I ran that command and I still get a glider. https://s11.postimg.org/fszcj3r1f/fgfs_screen_002.png I downloaded from the FG aircraft website last night and it worked properly. Perhaps it was the second to last commit on fgmembers as that is the one that I downloaded and got a glider. I'm not sure how adding smoke would break it but I also have not looked at it closely. I will SVN up and get the latest from FGadon to see if that one works properly. This will at least narrow it down.

**Comment by FGDATA** [comment]
Ok
I'll also try to narrow it down. but then certainly it is not the last command. 
My big suspect is a failure of the automatic merging from FGADDon. Maybe some files may show

            >>>HEAD
            *code hear
             ===SVN
             *other code here
             <<<<

Kind of structure. If so, I just need to perfect the merge and solve conflicts. I'll evaluate

**Comment by swampmo** [comment]
OK the "Mig-29" from FGAddon works properly so I would look at this commit in FGmembers. https://github.com/FGMEMBERS/MiG-29/commit/239643e2cbf0aca1ebe6d820583f62d9869790e3

**Comment by JWocky** [comment]
@swampmo: The 9-12 (Mercury Mercenary) produced a lot of Nasal errors in the electrical system here, even autostart doesn't work. So, maybe a difference in FG versions?

**Comment by swampmo** [comment]
Yes but they are two different models @JWocky I guess the long and short of it is it worked here some months ago when I downloaded it from FGmembers and now it doesn't. I only know because when I saw the changes "smoke" etc. I decided to try it last night.

**Comment by JWocky** [comment]
As I hinted on the forum, I won't fly her anyway at theUSA Tour, so if I need to do anything on any og the MiG-29s, someone needs to tell me, otherwise, I just play with my two planes on the short list for the Tour ;-)

**Comment by FGDATA** [comment]
@swampmo 

I think the problem is at your side

The model looks loading normal for me. FG 2017.1

the last commit I have says

              commit ea46a3ff593330b06635b3d2bff2b31b1adaa01a
              Merge: 0bedc85 239643e
              Author: FGDATA <fgdata.iahmcol@gmail.com>
              Date:   Tue Oct 18 17:03:52 2016 -0700
              
              Merge pull request #1 from FGMEMBERS/vezza-patch-1
              Adding missing effects

![fgfs-screen-392](https://cloud.githubusercontent.com/assets/10859459/20550537/7232baf8-b0eb-11e6-9538-fb95d484ca88.png)

I recommend you now verify that on the latest commit your content is clean

            git checkout master
            git status
